charts/signoz/templates/otel-collector/deployment.yaml

```diff
@@ -86,7 +86,7 @@ spec:
             {{- toYaml .Values.otelCollector.initContainers.init.resources | nindent 12 }}
         {{- end }}
       containers:
-        - name: {{ template "otelCollector.fullname" . }}
+        - name: app
```

Comment:
Any specific reason to name otel-collector container `app`?

Reply:
I wanted to go with the name `app` (application) for all main containers!